src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx

```diff
@@ -3105,6 +3105,16 @@ For more details, see our documentation about [APM logs in context](/docs/apm/ne
 </applicationLogging>
 ```
 
+These features can also be configured via environment variables:
+
+```
```

Comment:
@nr-ahemsath curious if you think it makes sense to include the values for these (="true") or if it'd be better to leave the values off if just giving examples.

Reply:
I was back and forth on it myself; in the end I decided to parallel the XML config file example shown immediately above the new section by having some concrete values.  I could go either way.